### ISSUE TITLE
Lua/Octopus Failure

### ISSUE THREAD

**Issue by nbrady-techempower** [TechEmpower/FrameworkBenchmarks#2557]
@cyberz-eu Octopus is failing again and it seems like the issue is again related to zlib removing a prior version https://github.com/cyberz-eu/octopus/blob/master/bin/unix/server.sh#L92

**Comment by knewmanTE** [comment]
zlib lives on github (https://github.com/madler/zlib/releases) so the ideal fix would be if octopus shifted over to using the github zips so we don't have this problem every time zlib is updated.

Also, I tested out adding a sed line to the `setup.sh` file to use zlib 1.2.11 and now I'm getting an error with the [database host](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Lua/octopus/config.lua#L24) line:

```
Setup octopus: /home/knewman/FrameworkBenchmarks/frameworks/Lua/octopus/octopus/bin/unix/luajit/bin/luajit: ../../extensions/config.lua:24: attempt to index global 'os' (a nil value)
Setup octopus: stack traceback:
Setup octopus: 	../../extensions/config.lua:24: in function 'f'
Setup octopus: 	../../extensions/core/src/eval.lua:41: in function 'file'
Setup octopus: 	build.lua:70: in main chunk
```

which points to the first line of this function in `eval.lua`:

```
local function evalFile(fileName, environment)
	local f = load_file(fileName, environment)
	local res, res2, res3, res4, res5 = f()
	return res, res2, res3, res4, res5
end
```

When I manually set the host attribute in `config.lua` to "127.0.0.1", all of the tests passed, but manually setting it to "TFB-database" didn't work and output the error:

```
...octopus/bin/unix/../../extensions/core/src/exception.lua:21: failed to connect: no resolver defined to resolve "TFB-database"
```

**Comment by nbrady-techempower** [comment]
@knewmanTE Yeah, that `sed` replacement was something I did the last time this came up and I submitted a PR to the Octopus repo to get it fixed, hopefully @cyberz-eu can nail down that issue. @msmith-techempower recently fixed the other two Lua frameworks that were having trouble with using the hosts aliases. I can look deeper into it tomorrow and see if any of the things he did could be applied here.

**Comment by msmith-techempower** [comment]
(This line)[https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Lua/lapis/config.lua#L17] is the relevant one for the Lua fixes.

**Comment by knewmanTE** [comment]
Temporary fix added with #2559, but would love a permanent, less-hacky solution.

**Comment by nbrady-techempower** [comment]
Going to close this for now. We've pinged the framework owner. If it comes up again, will re-open or remove the framework until fixed.
